s2n.mk

```diff
@@ -36,7 +36,12 @@ INDENT  = $(shell (if indent --version 2>&1 | grep GNU > /dev/null; then echo in
 ifeq ($(S2N_LIBCRYPTO), boringssl)
 	DEFAULT_CFLAGS = -std=c11
 else
-	DEFAULT_CFLAGS = -std=c99 -Wcast-qual
+	# AWS-LC is a BoringSSL derivative.
+	ifeq ($(S2N_LIBCRYPTO), awslc)
```

Comment:
Can use else-if conditional here to make this logic simpler: https://www.gnu.org/software/make/manual/html_node/Conditional-Syntax.html

Reply:
Changed.